apps/site/pages/dashboard/courseOverview.tsx

```diff
@@ -0,0 +1,182 @@
+import { GetServerSideProps } from "next";
+import { getSession } from "next-auth/react";
+import { EnrollmentDetails } from "@self-learning/types";
+import { getEnrollmentDetails } from "@self-learning/api";
+import React, { useState } from "react";
+import Link from "next/link";
+import { PlayIcon } from "@heroicons/react/24/solid";
+import { ProgressBar, Tab, Tabs } from "@self-learning/ui/common";
+
+export function CourseOverview({ enrollments }: { enrollments: EnrollmentDetails[] | null }) {
+	const [selectedTab, setSelectedTab] = useState(0);
+
+	if (!enrollments) {
+		return <p>No enrollments found</p>;
+	}
+
+	const inProgress = enrollments.filter(
+		enrollment => enrollment.completions.courseCompletion.completionPercentage < 100
+	);
+	const complete = enrollments.filter(
+		enrollment => enrollment.completions.courseCompletion.completionPercentage >= 100
+	);
+
+	return (
+		<div className="py-2 px-4">
+			{selectedTab === 0 && (
+				<TabContent
+					selectedTab={selectedTab}
+					setSelectedTab={setSelectedTab}
+					enrollments={inProgress}
+					notFoundMessage={"Derzeit ist kein Kurs angefangen."}
+				/>
+			)}
+			{selectedTab === 1 && (
+				<TabContent
+					selectedTab={selectedTab}
+					setSelectedTab={setSelectedTab}
+					enrollments={complete}
+					notFoundMessage={"Derzeit ist kein Kurs abgeschlossen."}
+				/>
+			)}
+		</div>
+	);
+}
+
+function EnrollmentOverview({
+	enrollments,
+	notFoundMessage
+}: {
+	enrollments: EnrollmentDetails[] | null;
+	notFoundMessage: string;
+}) {
+	if (!enrollments) {
+		return <p>Keine Kurse Gefunden.</p>;
+	}
+
+	return (
+		<div>
+			{enrollments.length > 0 ? (
+				<ul className="space-y-4">
+					{enrollments.map((enrollment, index) => (
+						<li
+							key={index}
+							className="flex flex-col space-y-2 rounded-lg bg-white p-4 shadow-md md:flex-row md:space-y-0 md:space-x-4"
+						>
+							{enrollment.course.imgUrl && (
+								<img
+									src={enrollment.course.imgUrl}
+									alt={enrollment.course.title}
+									className="h-24 w-full rounded-t-lg object-cover md:h-auto md:w-24"
+								/>
+							)}
```

Comment:
ich erhalte die warnung 

`Using `<img>` could result in slower LCP and higher bandwidth. Use `<Image />` from `next/image` instead to utilize Image Optimization. See: https://nextjs.org/docs/messages/no-img-elementeslint@next/next/no-img-element`